project/share/library_patches/arduino_mkrrgb/src/MKRRGBMatrix.h

```diff
@@ -0,0 +1,41 @@
+//
+// Created by danie on 2021-11-02.
+//
+
+#ifndef MKRRGBMATRIX_H
+#define MKRRGBMATRIX_H
+
+#include <cstddef>
+#include <cstdint>
+#include <ArduinoGraphics.h>
+#include <FramebufferAccess.h>
+#include "SMCE_dll.hpp"
```

Comment:
ideally these shoudnt go in the public header to avoid leaking stuff we dont want user to stumble upon.
you can forward declare FramebufferAccess in the header and then store it as a pointer or unique_ptr. this way the definition is hidden

Reply:
(And the class name should be prefixed with `SMCE__` so as to not conflict with user-defined identifiers)